packages/meditrak-server/src/tests/routes/userAccounts/GETUserAccounts.test.js

```diff
@@ -0,0 +1,184 @@
+    userAccount1 = await findOrCreateDummyRecord(models.user, {
+      first_name: 'Clark',
+      last_name: 'Kent',
+    });
+    userAccount2 = await findOrCreateDummyRecord(models.user, {
+      first_name: 'Bruce',
+      last_name: 'Wayne',
+    });
+    userAccount3 = await findOrCreateDummyRecord(models.user, {
+      first_name: 'Diana',
+      last_name: 'Prince',
+    });
+
+    // Give the test users some permissions
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount1.id,
+      entity_id: demoEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount1.id,
+      entity_id: kiribatiEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount2.id,
+      entity_id: demoEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount2.id,
+      entity_id: vanuatuEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount3.id,
+      entity_id: demoEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount3.id,
+      entity_id: kiribatiEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+    await findOrCreateDummyRecord(models.userEntityPermission, {
+      user_id: userAccount3.id,
+      entity_id: laosEntity.id,
+      permission_group_id: permissionGroup.id,
+    });
+
+    // Create a filter string so we are only requesting the test data from the endpoint
+    userAccountIds = [userAccount1.id, userAccount2.id, userAccount3.id];
+    filterString = `filter={"id":{"comparator":"in","comparisonValue":["${userAccountIds.join(
+      '","',
+    )}"]}}`;
+  });
+
+  afterEach(() => {
+    Authenticator.prototype.getAccessPolicyForUser.restore();
+  });
+
+  describe('GET /users/:id', async () => {
+    it('Sufficient permissions: Return a requested user account if we have access to all the countries the user has access to', async () => {
+      await prepareStubAndAuthenticate(app, DEFAULT_POLICY);
+      const { body: result } = await app.get(`users/${userAccount2.id}`);
+
+      expect(result.id).to.equal(userAccount2.id);
+    });
+
+    it('Sufficient permissions: Return a requested user account if we have BES admin access', async () => {
+      await prepareStubAndAuthenticate(app, BES_ADMIN_POLICY);
+      const { body: result } = await app.get(`users/${userAccount2.id}`);
+
+      expect(result.id).to.equal(userAccount2.id);
+    });
+
+    it('Insufficient permissions: Throw an exception if we do not have access to same countries as the user', async () => {
```

Comment:
Great test description here - can you use similarly specific descriptions in the edit tests?